### PR TITLE
Company attribute usage

### PR DESCRIPTION
**Pull request by dbelob**
Company attribute for speaker is added and used.

### CODE REVIEW

File: guess-game-server/src/main/java/guess/domain/source/Speaker.java [hunk 1]
```diff
@@ -43,12 +45,36 @@ public void setName(List<LocaleItem> name) {
...
+    public List<LocaleItem> getCompany() {
+        return company;
+    }
+
+    public void setCompany(List<LocaleItem> company) {
+        this.company = company;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```
Comment (inponomarev) [inline guess-game-server/src/main/java/guess/domain/source/Speaker.java]: I think Lombok is quite ok for this project

File: guess-game-server/src/main/java/guess/dto/guess/NamePicturesDto.java [hunk 2]
```diff
@@ -53,17 +56,31 @@ public String getFileName3() {
 
     public static NamePicturesDto convertToDto(String questionSetName, int currentIndex, int totalNumber, String logoFileName,
                                                QuestionAnswers questionAnswers, List<Long> wrongAnswerIds, Language language) {
+        Speaker questionSpeaker = ((SpeakerQuestion) questionAnswers.getQuestion()).getSpeaker();
```
Comment (inponomarev) [inline guess-game-server/src/main/java/guess/dto/guess/NamePicturesDto.java]: Looks like a copy-paste. Why not try to use e.g. arrays?

Comment (inponomarev) [reply guess-game-server/src/main/java/guess/dto/guess/NamePicturesDto.java]: There is already a lot of code like this in the project, though

File: guess-game-server/src/main/java/guess/domain/source/Speaker.java (hunk 1 above)
Comment (dbelob) [reply guess-game-server/src/main/java/guess/domain/source/Speaker.java]: I don't like Lombok.

File: guess-game-server/src/main/java/guess/dto/guess/NamePicturesDto.java (hunk 2 above)
Comment (dbelob) [reply guess-game-server/src/main/java/guess/dto/guess/NamePicturesDto.java]: You can try refactoring in the branch and show the result. The application should remain functional, of course.

**Pull request merged**